eng/scripts/Remove-PRBranch.ps1

```diff
@@ -0,0 +1,29 @@
+param(
+    [string] $Repo,
+    [string] $CommitId,
+    [string] $BodyFile
+)
+
+if (Test-Path $BodyFile) {
+    $repomanContent = Get-Content $BodyFile -Raw | ConvertFrom-Json
+    $itemCount = $repomanContent.count
+    $PRNumber = gh api "/repos/$Repo/commits/$CommitId/pulls"  | jq -r '.[].number'

```

Comment:
Just to confirm, this will only delete the PR branches from the commit/PR being merged, correct?

Reply:
yes those pull request branches that got created as part of the pull request will be removed once the pull request is merged

Reply:
Yes, to confirm, only the PR branch from the merge is deleted . Other branches will remain in place on the target template repo.